src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
     <PropertyGroup>
-        <VersionPrefix>7.0.1</VersionPrefix>
+        <VersionPrefix>7.0.2</VersionPrefix>
```

Comment:
@nunorelvao , I will merge this PR as-is. But please don't change the version. I will do it and generate the new version.